Not installing on Windows 10 with VS2017

When trying to install the module (as an optional dependency from another one) on a machine with Windows 10 and Visual Studio 2017 installed, I'm getting the following build error:
`MSBUILD : error MSB4025: The project file could not be loaded. Root element is missing.`

Here's the full output from npm:
```
> sse4_crc32@5.1.1 install <path to project>\node_modules\sse4_crc32
> node-gyp rebuild


<path to project>\node_modules\sse4_crc32>if not defined npm_config_node_gyp (node "D:\nvm\v6.11.5\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB4025: The project file could not be loaded. Root element is missing.
gyp ERR! build error
gyp ERR! stack Error: `msbuild` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (D:\nvm\v6.11.5\node_modules\npm\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:219:12)
gyp ERR! System Windows_NT 10.0.16299
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "D:\\nvm\\v6.11.5\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd <path to project>\node_modules\sse4_crc32
gyp ERR! node -v v6.11.5
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: sse4_crc32@5.1.1 (node_modules\sse4_crc32):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: sse4_crc32@5.1.1 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
```

As far as I read the other Windows-related issues this problem is not mentioned, so I'm assuming it's a new one. Shall I try to manually upgrade the project file by opening it with VS2017 or something else?

Note 1: I'm trying to build using "x86_x64 Cross Tools Command Prompt for VS 2017" (which supposedly should have the proper environment setup).
Note 2: My Node.js is installed via nvm-windows.

Thank you in advance!

--- a/package.json
+++ b/package.json
@@ -26,13 +26,13 @@
   "main": "./sse4_crc32",
   "dependencies": {
     "bindings": "~1.2.1",
-    "nan": "^2.3.5"
+    "nan": "2.8.0"
   },
   "devDependencies": {
-    "node-gyp": "^3.3.1",
-    "mocha": "^2.1.0",
     "chai": "^2.0.0",
-    "crc32": "^0.2.1"
+    "crc32": "^0.2.1",
+    "mocha": "^2.1.0",
+    "node-gyp": "^3.3.1"
   },
   "scripts": {
     "test": "mocha test/*test.js",

--- a/src/crc32c.cpp
+++ b/src/crc32c.cpp
@@ -42,9 +42,11 @@ static uint32_t crc32cTable[8][256];
 /**
  * Cross-platform CPU feature set detection to check for availability of hardware-based CRC-32C
  */
+#ifdef _MSC_VER
+#include <intrin.h> // __cpuid
+#endif
 void cpuid(uint32_t op, uint32_t reg[4]) {
-#if defined(_WIN64) || defined(_WIN32)
-    #include <intrin.h>
+#if defined(_MSC_VER)
     __cpuid((int *)reg, 1);
 #elif defined(__x86_64__)
     __asm__ volatile(

